### DISCUSSION TITLE
Package wheel after add some folders/files using pyproject.toml

### DISCUSSION THREAD

**Discussion by Yezhj1** [pypa/setuptools#3604] [Q&A]
I want to package my python project to wheel using pyproject.toml, and run `python3 -m build --wheel`. Before i run `python3 -m build --wheel`, i need to add some folder/files into my project, then i run `python3 -m build --wheel`, and get a wheel, but i can not find the new folders/files in the wheel. 
help!!!!!!!!

**Comment by abravalheri** [comment]
Hi @Yezhj1, you might want to review the information [in this doc page](https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html) about how to control the files in the distribution.

**Reply by abravalheri** [reply]
Please not however, that as rule of thumb, for a file to be "easily" included in the `wheel`, it should also be present in the `sdist`.
